projects/npm-tools/packages/npm-scripts/src/config/npmscripts.config.js

```diff
@@ -3,8 +3,143 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-var path = require('path');
+const path = require('path');
+
+const CHECK_AND_FIX_GLOBS = [
```

Comment:
These are the globs I was referrring to in [this comment](https://github.com/liferay/liferay-frontend-projects/pull/420#discussion_r582933129)